### PR TITLE
C interface/Python: Getter for numerical values

### PR DESCRIPTION
**Pull request by sandmaennchen**
This PR adds getters for numerical values in cost and constraint modules and interfaces them from Python.

Adds the following functions to `AcadosOcpSolver`:
- `cost_get(stage, field)`
- `constraints_get(stage, field)`

### CODE REVIEW

File: examples/acados_python/max_iter_test/max_iter_test.py [hunk 1]
```diff
@@ -106,6 +106,13 @@ def create_acados_solver_and_solve_problem(globalization='FIXED_STEP', solver_ty
 
     status = ocp_solver.solve()
 
+    # test getter
+    Vu_ = ocp_solver.cost_get(1, "Vu")
+    Vx_ = ocp_solver.cost_get(1, "Vx")
```
Comment (david0oo) [inline examples/acados_python/max_iter_test/max_iter_test.py:111]: Can we test the other cost getters that were newly added as well?

Comment (david0oo) [reply examples/acados_python/max_iter_test/max_iter_test.py:111]: I think we need an example that has an external cost and an example with NLS.

File: examples/acados_python/pendulum_on_cart/ocp/example_sqp_rti_loop.py [hunk 2]
```diff
@@ -145,6 +145,9 @@ def main():
     cost = ocp_solver.get_cost()
     print("cost function value of solution = ", cost)
 
+    # test getter
```
Comment (david0oo) [inline examples/acados_python/pendulum_on_cart/ocp/example_sqp_rti_loop.py:148]: Same for constraints_get? Can we add more tests for all the new getters?

**Review by david0oo — CHANGES_REQUESTED** [review]

File: examples/acados_python/pendulum_on_cart/ocp/example_sqp_rti_loop.py (hunk 2 above)
Comment (sandmaennchen) [reply examples/acados_python/pendulum_on_cart/ocp/example_sqp_rti_loop.py:148]: will be tested by evaluator :smile:, cf https://github.com/acados/acados/pull/1402

**Pull request merged**